Pod/Classes/URBNAccordionDataSourceAdapter.m

```diff
@@ -117,6 +117,22 @@ - (BOOL)sectionIsOpen:(NSInteger)section {
     return [self.sectionsToKeepOpen containsIndex:section] || [self.expandedSections containsIndex:section];
 }
 
+- (void)appendSectionWithItems:(NSArray *)newItems {
+    NSAssert(YES, @"Call appendSectionObject:items:expanded: instead");
+}
+
+- (void)appendSectionObject:(id)sectionObject items:(NSArray *)items {
+    self.sections = [self.sections arrayByAddingObject:sectionObject];
```

Comment:
You should make sure that items is an actual array. You'll see in the initWithItems method we `arrayWithArray:items`. That way its A) not mutable and B) not inserting nil into the containing array